nat-lab/tests/utils/vm/mac_vm_util.py

```diff
@@ -78,6 +79,7 @@ async def _copy_binaries(
             f"{VM_UNIFFI_DIR}libtelio_remote.py",
             True,
         ),
+        (f"{LOCAL_BIN_DIR}multicast.py", f"{VM_TCLI_DIR}multicast.py", True),
```

Comment:
This file is copied to the mac VM, but how is it being used? I'm not seeing any mention of running it in this review.